backend/prompt_studio/prompt_studio_v2/serializers.py

```
@@ -6,10 +6,59 @@
+                from lookup.models import PromptStudioLookupLink
+
+                link_exists = PromptStudioLookupLink.objects.filter(
+                    prompt_studio_project_id=tool_id,
+                    lookup_project=value,
+                ).exists()
+
+                if not link_exists:
+                    raise serializers.ValidationError(
+                        "Selected lookup project must be linked to this "
+                        "Prompt Studio project at the project level first."
+                    )
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)